packages/docusaurus-theme-mermaid/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@docusaurus/theme-mermaid",
+  "version": "2.0.0-beta.20",
+  "description": "Mermaid components for Docusaurus.",
+  "main": "lib/index.js",
+  "exports": {
+    "./theme/MDXComponents/Mermaid": "./lib/theme/MDXComponents/Mermaid.js",
+    "./theme/useMermaid": "./lib/theme/useMermaid.js",
+    "./theme/Mermaid": "./lib/theme/Mermaid/index.js",
```

Comment:
Don't do this—theme paths will be registered by the plugin. Nothing should import theme from the module directly, because this makes swizzling impossible.

Reply:
Done